.travis.yml

```diff
@@ -33,6 +33,8 @@ virtualenv:
 install:
   - pip install requests
   - pip install unicodecsv
+  - pip install python-stdnum==1.1
```

Comment:
What about use the `requirements.txt` file in root path of the project?
`partner-contact/requirements.txt`
MQT auto-process this file more info: https://github.com/OCA/maintainer-quality-tools/pull/311